### PR TITLE
feat: add openapi schema diff function

### PR DESCRIPTION
**Pull request by ivan-tymoshenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+- [Installation](#installation)
+
+<a name="installation"></a>
+
+## Installation
+
+```bash
+npm install openapi-schema-diff
+```
```
Comment (mcollina) [inline README.md:17]: Add usage and license section

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,3489 @@
+{
```
Comment (mcollina) [inline package-lock.json]: ignore this file

File: test/operation-objects.test.js [hunk 3]
```diff
@@ -0,0 +1,848 @@
...
+            before: 'integer',
+            after: 'string'
+          }
+        ]
+      }
+    ]
+  })
+})
+
+test('should not count summery and description properties', async () => {
```
Comment (mcollina) [inline test/operation-objects.test.js]: I don't think these needs to be async functions

File: index.js [hunk 4]
```diff
@@ -0,0 +1,241 @@
+'use strict'
+
+const { randomUUID } = require('node:crypto')
+const { major: semverMajor } = require('semver')
+const { RefResolver } = require('json-schema-ref-resolver')
+const { compareObjectKeys } = require('./lib/utils.js')
+
+const HTTP_METHODS = ['get', 'put', 'post', 'delete', 'options', 'head', 'patch', 'trace']
```
Comment (mcollina) [inline index.js:8]: It's likely better to use Node.js core list of methods, or maybe this is limited for a reason?

Comment (ivan-tymoshenko) [reply index.js:8]: this is a list from openapi specification
https://swagger.io/specification/#path-item-object

File: package.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  "scripts": {
+    "lint": "standard",
+    "lint:fix": "standard --fix",
+    "test:unit": "c8 --100 node --test",
+    "test": "npm run lint && npm run test:unit"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fastify/openapi-schema-diff.git"
+  },
```
Comment (Eomm) [inline package.json:15]:
```suggestion
  },
    "bugs": {
    "url": "https://github.com/fastify/openapi-schema-diff/issues"
  },
  "homepage": "https://github.com/fastify/openapi-schema-diff#readme",
```

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**